tests/test_observables.py

```diff
@@ -534,6 +535,16 @@ def test_mb_get(self, mock_get=None, mock_post=None):
         ).start()
         self.assertEqual(report.get("success", False), True)
 
+    def test_mb_google(self, mock_get=None, mock_post=None):
```

Comment:
this should be put in `utils.py` under the `CommonTestCases_ip_url_domain` class